From 64e358821e0e8b5a2134aa93ba9ddf24cd46ac7c Mon Sep 17 00:00:00 2001
From: Oleksandr Byelkin <sanja@mariadb.com>
Date: Fri, 4 Feb 2022 14:52:03 +0100
Subject: [PATCH] Revert "don't build with OpenSSL 3.0, it doesn't work before
 MDEV-25785"

This reverts commit c9beef43154a199bfcd9f71049c011a2ed77ca74, because
we have OpenSSL 3.0 support here.
diff -wpruN --no-dereference '--exclude=*.orig' a~/cmake/ssl.cmake a/cmake/ssl.cmake
--- a~/cmake/ssl.cmake	1970-01-01 00:00:00
+++ a/cmake/ssl.cmake	1970-01-01 00:00:00
@@ -118,7 +118,7 @@ MACRO (MYSQL_CHECK_SSL)
     ENDIF()
     FIND_PACKAGE(OpenSSL)
     SET_PACKAGE_PROPERTIES(OpenSSL PROPERTIES TYPE RECOMMENDED)
-    IF(OPENSSL_FOUND AND OPENSSL_VERSION AND OPENSSL_VERSION VERSION_LESS "3.0.0")
+    IF(OPENSSL_FOUND)
       SET(OPENSSL_LIBRARY ${OPENSSL_SSL_LIBRARY})
       INCLUDE(CheckSymbolExists)
       SET(SSL_SOURCES "")
